pagopa/io-app
# fix: [IOBP-134] Add navigation to initiative details after IDPay transaction authorization

## Short description
This PR adds the navigation to the initiative details screen after IDPay payment authorization.

https://github.com/pagopa/io-app/assets/6160324/3bc2de1d-c7f8-41ce-9481-f42cd23a6b31



## List of changes proposed in this pull request
- [ts/features/idpay/payment/xstate/actions.ts](https://github.com/pagopa/io-app/compare/IOBP-134-fix-landing-after-authorization?expand=1#diff-130a75c56f991ea3cc1e3857a98c902845fe2d9f1cc7fc10b17e98fd24769557):  added navigation to details screen on authorization exit

## How to test
With the `io-dev-server`, checkout the branch from [this PR](https://github.com/pagopa/io-dev-api-server/pull/283) and  authorize a payment and check that the app navigates to the details screen after the authorization process.
